app/templates/admin_template.html

```diff
@@ -1,6 +1,5 @@
 {% extends "main_template.html" %}
 {% from "components/banner.html" import banner %}
-{% from "components/cookie-banner.html" import cookie_banner %}
```

Comment:
[suggestion] is this a file that can be removed? Or do we want to keep it around in case we add that back?